frontend/src/main.ts

```diff
@@ -1,8 +1,44 @@
-import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { importProvidersFrom } from '@angular/core';
+import { AppComponent } from './app/app.component';
+import { MarkdownModule, MarkedOptions } from 'ngx-markdown';
+import { routes } from './app/app.routes';
+import { provideRouter, withInMemoryScrolling } from '@angular/router';
+import { bootstrapApplication } from '@angular/platform-browser';
+import { errorInterceptor } from './app/error-interceptor.service';
+import { HttpClient, provideHttpClient, withInterceptors } from '@angular/common/http';
+import { markedOptionsFactory } from './marked-options.factory';
+import { provideI18n } from './app/i18n/i18n';
 
-import { AppModule } from './app/app.module';
-
-platformBrowserDynamic()
-  .bootstrapModule(AppModule)
+bootstrapApplication(AppComponent, {
+  providers: [
+    importProvidersFrom(
+      MarkdownModule.forRoot({
+        loader: HttpClient,
+        markedOptions: {
+          provide: MarkedOptions,
+          useFactory: markedOptionsFactory,
+          useValue: {
+            gfm: true,
+            tables: true,
+            breaks: false,
+            pedantic: false,
+            sanitize: false,
+            smartLists: true,
+            smartypants: false
+          }
+        }
+      })
+    ),
+    provideRouter(
+      routes,
+      withInMemoryScrolling({
+        scrollPositionRestoration: 'disabled',
```

Comment:
nit: there was a comment explaining why we disabled it:
```
// when enabled, every click on an aggregation scrolls the page to the top, which is annoying
```
I think we should leave it, to remember why we disabled it.